Jenkinsfile

```diff
@@ -889,7 +889,8 @@ pipeline {
                             label 'docker_runner'
                             additionalBuildArgs dockerBuildArgs(repo_type: 'stable',
                                                                 parallel_build: true,
-                                                                deps_build: true)
+                                                                deps_build: true) +
+                                                "--build-arg BASE_DISTRO=centos:8"
```

Comment:
This seems like it should be returned with all of the other args that `dockerBuildArgs()` returns.

Reply:
There's plenty of code that does this already, and that would involve a change to pipeline-lib.  Maybe in the future dockerBuildArgs could do this automatically based on the stageInfo information?

Reply:
> There's plenty of code that does this already

In the Jenkins pipeline?  I can't see any other instance where `--build-arg BASE_DISTRO` is being set.

> that would involve a change to pipeline-lib

Indeed, it would.  But it's the right place to deal with it if we are going to forward on the use of `BASE_DISTRO` in our Dockerfiles (which we should, so that we have one Dockerfile that covers all EL 8 distros) so that it's done in one place only and not duplicated as more use of the EL 8 Dockerfile is introduced, which will happen soon enough as more build and test in Jenkins moves to EL 8.